rum-demo-app-ios/RUM Mobile Demo App/Assets/Local HTMLs/index.html

```diff
@@ -0,0 +1,48 @@
+<html>
+    <meta name="viewport" content="width=device-width, shrink-to-fit=YES">
+        <head>
+            <!-- Tip: Use relative URLs when referring to other in-app content to give
+            your app code the flexibility to change the scheme or domain as
+            necessary. -->
+            <link rel="stylesheet" href="stylesheet.css">
+                <script src="https://cdn.signalfx.com/o11y-gdi-rum/latest/splunk-otel-web.js"
+                    crossorigin="anonymous"></script>
+                <script type="text/javascript">
+                    function initialiseRUM(data) {
+                        var jsons = JSON.parse(data)
+                        SplunkRum.init({
+                            beaconUrl: 'https://rum-ingest.' + jsons.realm  + '.signalfx.com/v1/rum',
+                            rumAuth: jsons.token,
+                            app: "iOS WebView Spectacular.com"
```

Comment:
@akashCrest Let's rename the app here as "RUM Demo App Webview"

Reply:
Done